src/content/resources/language/evolution.md

```diff
@@ -42,6 +42,36 @@ on the Dart language GitHub repo.
 
 ## Changes in each release
 
+### Dart 3.8
+_Released 20 May 2025_
+| [Dart 3.8 announcement](https://medium.com/dartlang/...)
+
+The following language features have been added to Dart 3.8:
+
+*   [Null-aware elements][]: A null-aware element evaluates
+    an expression and if the result is not null, inserts the
+    value into the surrounding collection.
```

Comment:
Thanks for adding this and cross-linking to your fantastic docs about it!

Consider clarifying earlier in the sentence that this is discussing a collection element, so readers can contextualize it.

```suggestion
*   [Null-aware elements][]: A null-aware collection element evaluates
    an expression in a collection literal and if the result is not `null`,
    inserts the value into the resulting collection.
```